src/app/public/modules/inline-delete/inline-delete.component.ts

```diff
@@ -119,6 +126,22 @@ export class SkyInlineDeleteComponent implements OnDestroy, OnInit {
    */
   public ngOnDestroy(): void {
     this.adapterService.clearListeners();
+    this.cancelTriggered.complete();
+    this.deleteTriggered.complete();
+  }
+
+  /**
+   * @internal
+   */
+  public onCancelClick(): void {
+    this.animationState = 'hidden';
+  }
+
+  /**
+   * @internal
+   */
+  public onDeleteClick(): void {
+    this.deleteTriggered.emit();
```

Comment:
This method (and the one above it) house logic that was pulled out of the template, to make the logic easier to read and test.